Classes/NodeCreationHandler/NodeCreationCommands.php

```diff
@@ -0,0 +1,58 @@
+<?php
+declare(strict_types=1);
+
+namespace Neos\Neos\Ui\NodeCreationHandler;
+
+use Neos\ContentRepository\Core\CommandHandler\CommandInterface;
+use Neos\ContentRepository\Core\Feature\NodeCreation\Command\CreateNodeAggregateWithNode;
+use Neos\ContentRepository\Core\Feature\NodeModification\Dto\PropertyValuesToWrite;
+
+class NodeCreationCommands implements \IteratorAggregate
```

Comment:
i dislike `CreateNodeAggregateWithNodeAndAdditionalCommands`

thats the one i like `NodeAggregateWithNodeCreationRecipe`